Sources/Slacket/Models/SlacketModel.swift

```diff
@@ -0,0 +1,18 @@
+//
+//  SlacketModel.swift
+//  Slacket
+//
+//  Created by Jakub Tomanik on 20/05/16.
+//
+//
+
+import Foundation
+
+protocol SlacketModel {}
```

Comment:
I was thinking it would be good to have some kind of "base" protocol even if empty that all models would implement, jus to distinguish them.